docs/products/mysql/howto/migrate-from-external-mysql.md

```diff
@@ -1,84 +1,102 @@
+To migrate your database using the Aiven Console, see
+[Migrate MySQL® databases to Aiven via console](/docs/products/mysql/howto/migrate-db-to-aiven-via-console).
 :::
 
-The process will first do a `mysqldump` to seed the schema and bulk-copy the
-data, if the preconditions are met for ongoing replication then it will
-configure MySQL as a replica of the external database.
+## How it works
 
-## Requirements
+The Aiven for MySQL migration process begins with an **initial data transfer** and can be
+followed by **continuous data synchronization** if your setup supports it.
 
-To perform a migration from an external MySQL to Aiven for MySQL the
-following requirements need to be satisfied:
+### Initial data transfer
 
--   The source server needs to be publicly available or accessible via a
-    virtual private cloud (VPC) peering connection between the private
-    networks, and any firewalls need to be open to allow traffic between
-    the source and target servers.
--   You have a user account on the source server with sufficient
-    privileges to create a user for the replication process.
--   [GTID](https://dev.mysql.com/doc/refman/8.0/en/replication-gtids.html)
-    is enabled on the source database. To review the current GTID
-    setting, run the following command hon the source cluster:
+A bulk copy of your data is first created. This is done using one of the following tools:
 
-    ```bash
-    show global variables like 'gtid_mode';
-    ```
+- `mysqldump` for small and medium-sized databases
+- `mydumper/myloader` <EarlyBadge/> for large databases
+
+<MydumperNote/>
+
+### Continuous data synchronization
+
+After the initial data copy, the Aiven for MySQL service can be configured as a replica of
+your external database, enabling ongoing data synchronization through MySQL's built-in
+replication feature.
+
+## Prerequisites
+
+- The source server is publicly available or accessible via a virtual private cloud (VPC)
+  peering connection between the private networks, and firewalls are open to allow traffic
+  between the source and target servers.
+- You have a user account on the source server with sufficient privileges to create a user
+  for the replication process.
+- [GTID](https://dev.mysql.com/doc/refman/8.0/en/replication-gtids.html) is enabled on the
+  source database. Review the current GTID setting by running the following command on the
+  source cluster:
+
+  ```bash
+  show global variables like 'gtid_mode';
+  ```
+
+- You have a running Aiven for MySQL service with a destination database. If missing,
+  create it in the [Aiven Console](/docs/products/mysql/get-started) or the
+  [Aiven CLI](/docs/tools/cli/service-cli#avn-cli-service-create).
+
+- If you use `mydumper/myloader` for the
+  [initial data transfer](/docs/products/mysql/howto/migrate-from-external-mysql#initial-data-transfer),
+  make sure your service has significant computational power (multiple vCPUs) and high
```

Comment:
```suggestion
  make sure your service has sufficient computational power (multiple vCPUs) and high
```
how about sufficient instead of significan, to indicate the service needs enough computational power for migration.